plonk-verify.cabal

```diff
@@ -37,65 +17,44 @@ version:            0.1.0
 -- patches.
 -- maintainer:
 
--- A copyright notice.
--- copyright:
+copyright: Copyright (c) 2024 Argument Computer Corporation
 
-extra-source-files: Cargo.toml Cargo.lock src/lib.rs
+extra-doc-files:
+    README.md
 
--- This let us hook Cabal steps to Setup.lhs script.
-build-type:         Custom
-custom-setup
-    setup-depends:  Cabal, base, directory, process
+extra-source-files:
+    Cargo.toml
+    Cargo.lock
+    src/lib.rs
+    hsbindgen.toml
 
--- Extra doc files to be distributed with the package, such as a CHANGELOG or a
--- README.
--- extra-doc-files:
+data-files:
+    verifier-assets/v1.0.8-testnet/vk.bin
```

Comment:
Note: even though the `vk.bin` being listed here makes sense, I'm not sure how/if it can be accessed from other cabal packages with this method alone (specially if the library is not installed but simply used as a dependency). Is this enough to make it accessible even from a dependency-only context by itself?